src/core/components/hv-element/index.tsx

```diff
@@ -0,0 +1,157 @@
+import * as InlineContext from 'hyperview/src/services/inline-context';
+import * as Logging from 'hyperview/src/services/logging';
+import * as Namespaces from 'hyperview/src/services/namespaces';
+import { LOCAL_NAME, NODE_TYPE } from 'hyperview/src/types';
+import React, { useMemo } from 'react';
+import type { HvComponentProps } from 'hyperview/src/types';
+
+export default (
+  props: HvComponentProps,
+): React.ReactElement<HvComponentProps> | null | string => {
```

Comment:
can the return value be
```suggestion
): JSX.Element | null => {
```

Reply:
https://github.com/Instawork/hyperview/pull/1169/commits/8df786f2be690b598e2b5d5c05f93f69276b8fae